### ISSUE TITLE
When setting a tag using the `tag,color` syntax, removing that tag fails unless the color is also specified.

### ISSUE THREAD

**Issue by luckman212** [RhetTbull/osxmetadata#106]
### Config

- osxmetadata v1.3.4 (installed in a venv via pip)
- macOS 14.5

### Summary

When setting a tag using the `tag,color` syntax, removing that tag fails unless the color is also specified.

### Example

```
osxmetadata --set tags Foo,red /path/to/quux.txt
```

Result: _`quux.txt` successfully tagged with a red **Foo**_

```
osxmetadata --remove tags Foo /path/to/quux.txt
```

Result: _no error, but the **Foo** tag is still present!_ ❌

```
osxmetadata --remove tags Foo,red /path/to/quux.txt
```

Result: **Foo tag successfully removed** ✅

### Conclusion

Removing a tag should not require the optional color to be specified.

**Comment by RhetTbull** [comment]
Thanks. I'll look into this when I have time. Am busy on another project at the moment.

**Comment by RhetTbull** [comment]
@all-contributors please add @luckman212 for bugs

**Comment by RhetTbull** [comment]
Looking into this. Colored named tags are very problematic. Personally I only use colors with the color name (e.g. "Red") because of the way Finder handles these. For example, if you previously created a tag `Foo,blue` then you set the tag `Foo,red`, the tag would be set with color `blue` because Finder already has that cached.

I think I can use `get_finderinfo_color()` to get the color Finder associates with the tag in the case there is no color assigned then use that to remove the tag. May be a little while before I can get to this.

**Comment by luckman212** [comment]
No worries. Very low priority! Appreciate the 👀  
Enjoy your summer.

**Comment by RhetTbull** [comment]
This should be fixed in v1.3.5

**Comment by luckman212** [comment]
Confirmed - indeed it is! 🚀 
Much appreciated, thanks for this.
